### ISSUE TITLE
Add top level language selection

### ISSUE THREAD

**Issue by dgershman** [bmlt-enabled/bmlt-api-explorer#9]
Not sure if you considered this, but the original version of this was in English only.  It might make sense to add a language dropdown and create resource files ahead of time, in anticipation that this would be translated into multiple languages.

**Comment by GarrettWeinberg** [comment]
@radius314 im looking into solutions for this. I did find a couple i18n packages that would probably be suitable to make this work. I dont have much experience working with translations outside of wordpress, so i am open to any insight or suggestions on how to handle this. Also, is it safe to assume that the data that is returned from the server will already be translated?

**Comment by dgershman** [comment]
Yes it would be safe to assume that, we'd pass in the appropriate language
code to each query to get that data.
